src/rpcdump.cpp

```diff
@@ -246,10 +246,21 @@ Value dumpprivkey(const Array& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "dumpprivkey <Lindaaddress>\n"
-            "Reveals the private key corresponding to <Lindaaddress>.");
+            "Reveals the private key corresponding to <Lindaaddress>.\n"
+            "\nDo not recklessly execute this command in front of others, \n"
+            "hackers/scammers are known to use this method to steal your coins!");
 
     EnsureWalletIsUnlocked();
 
+    static int count = 0;   /*  Display a warning the first time a user executes this command */
```

Comment:
This variable will be re-set every time this method is hit so the user will always be shown the message

Reply:
Fixed